requests/sessions.py

```diff
@@ -164,6 +164,14 @@ def resolve_redirects(self, resp, req, stream=False, timeout=None,
             proxies = self.rebuild_proxies(prepared_request, proxies)
             self.rebuild_auth(prepared_request, resp)
 
+            # Attempt to rewind consumed file-like object
```

Comment:
I think we need to expressly extend this comment to talk about why we want to allow the `object()` from `models.py` to mark the body `rewindable`.